### ISSUE TITLE
Args: `--version/-V` was missing

### ISSUE THREAD

**Issue by Pennyzct** [firecracker-microvm/firecracker#1632]
Hi guys
I've found that in new release Firecracker `v0.21.0`, the command line arg `--version/-V` is missing.
```
$ ./firecracker-v0.21.0-x86_64 --version
2020-02-26T03:15:13.135125190 [anonymous-instance:ERROR:src/firecracker/src/main.rs:132] Arguments parsing error: Found argument 'version' which wasn't expected, or isn't valid in this context.
```
Checking out the existing flags through `--help`, see `--version/-V` is also not there. But `--help` will 
display the version info in the first line. 
```
Firecracker v0.21.0

--api-sock <api-sock>: Path to unix domain socket used by the API.
--id <id>: MicroVM unique identifier.
--help: Show the help message.
--no-api: Optional parameter which allows starting and using a microVM without an active API socket.
--start-time-us <start-time-us>
--seccomp-level <seccomp-level>: Level of seccomp filtering that will be passed to executed path as argument.

                        - Level 0: No filtering.

                        - Level 1: Seccomp filtering by syscall number.

                        - Level 2: Seccomp filtering by syscall number and argument values.

```
So `--version/-V` has already been deprecated in the new release??

**Comment by Pennyzct** [comment]
I've found that in PR https://github.com/firecracker-microvm/firecracker/pull/1360, we've replaced clap with our own args parser, and maybe forgot to add the dealing with `--version/-V`.

**Comment by dianpopa** [comment]
Solved by patch release #1654.
